src/lib.rs

```diff
@@ -580,6 +580,18 @@ pub mod tests {
         Field, serialize::test_canonical_serialize_deserialize, SemanticallyValid
     };
     use rand::{distributions::Distribution, Rng, thread_rng};
+    use crate::rng::FiatShamirRngSeed;
+
+    fn get_test_fs_rng<F, PC>() -> PC::RandomOracle
```

Comment:
Not important: Why don't we call it ``setup_test_fs_rng``?

Reply:
Done